public/message.ts

```diff
@@ -0,0 +1,63 @@
+import { updateScroll } from "./ui";
+import { gameOver } from "./game";
+import { getLogsElement, getElementById } from "./helpers";
+
+// TODO: What type to use for message - string or MessageEvent?
+export const handleMessage = (message: MessageEvent): void => {
+  // string type introduces errors
+  let msg = JSON.parse(message.data);
+  console.log(msg);
+  let currentText = getLogsElement().innerHTML;
+  switch (msg.type) {
+    case "prompt":
```

Comment:
```suggestion
    case "prompt": {
```

Reply:
(if it complains about using the same variable name multiple times, wrapping the case arm in a {} block will give you a new scope, so the variables can't collide)